code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -222,6 +222,16 @@ LINEN BINS
 	item_color = "ian"
 	dream_messages = list("a dog", "a corgi", "woof", "bark", "arf")
 
+/obj/item/bedsheet/cosmos
+	name = "cosmic space blankie"

```

Comment:
These are all called bedsheets, not blankets. I still don't know what these words mean in English, but I think it's better to stick to what's already there.